app/components/rdf-form-fields/property-group-selector.js

```diff
@@ -46,7 +46,13 @@ export default class PropertyGroupSelector extends InputFieldComponent {
         );
         return { subject: t.subject, label: label && label.value };
       });
-    this.options.sort(byLabel);
+
+    // There are option labels that are undefined in this list, I did not found it yet why or where these values are added but if they do not have a label they should not be visible in the frontend
+    const optionsWithALabel = allRawOptions.filter((_option) => {
```

Comment:
Nitpick: Why the `_undescored` variable names? 😄

Reply:
Thought:
This was done so it's clear that it is a parameter in a smaller scope than the main code. The variable option could be used in the main code and in the filter method. This also only visually because it would be using `this.option` but on the other side when passing arguments like in the refresh task here we can use one of these without the this keyword this `*refresh({ value, resetBuilder, isInitialRouteCall = false }) {`

Not sure how to handle those differences? Should we just use the parameter without the underscore?